Examples don't compile

Both of the examples fail with:
```
error[E0308]: mismatched types
   --> src/lib.rs:167:1
    |
167 | ngx_modules!(ngx_http_hello_world_module);
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `i8`, found `u8`
    |
    = note: expected raw pointer `*const i8`
               found raw pointer `*const u8`
    = note: this error originates in the macro `ngx_modules` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0308`.
error: could not compile `hello-rs` due to previous error
```

--- a/nginx-rs/nginx-sys/src/lib.rs
+++ b/nginx-rs/nginx-sys/src/lib.rs
@@ -18,7 +18,7 @@ macro_rules! ngx_modules {
 
         #[no_mangle]
         pub static mut ngx_module_names: [*const c_char; $crate::count!($( $mod, )+) + 1] = [
-            $( concat!(stringify!($mod), "\0").as_ptr() as *const u8, )+
+            $( concat!(stringify!($mod), "\0").as_ptr() as *const c_char, )+
             ptr::null()
         ];
 

